src/recaplog

```diff
@@ -89,7 +89,7 @@ delete_old_logs() {
   matched_files=$(
     find "${BASEDIR}" -maxdepth 2 -regextype posix-extended \
       -type f -mtime +${LOG_EXPIRY} -regex \
-      "^${BASEDIR}/(.*/)?[a-z]+_([a-z]+_)?[0-9]{8}(-[0-9]{6})?\.log(\.tar)?(\.gz)?$"
+      "^${BASEDIR}/(.*/)?[a-z]+_([a-z]+_)*[0-9]{8}(-[0-9]{6})?\.log(\.tar)?(\.gz)?$"
```

Comment:
Can you explain what this update to the regex is doing (in real life)? We're going from a single character match to a greedy wildcard, it may match more that expected? (meaning does it need to be non-greedy, like `.*?` to ensure the RHS of it doesn't get sucked into the greed?)

Reply:
The important change is
From:
  ```
  ([a-z]+_)?
  ```
To:
  ```
 ([a-z]+_)*
  ```

In the past we did only have logs as these examples:
- `ps_20181112-111200.log`
- `ps_daily_20181028.log.tar.gz`

With `http_status` and any  other plugin using more than one single word the format will be (examples):
- `http_status_20181112-111200.log`
- `http_status_daily_20181112-111200.log.tar.gz`

Hence the change. Hope that helps to clarify that change.

Reply:
gotcha! I fired up <https://www.debuggex.com/> and played a bit with random input, matches expectations. +1